s3path/current_version.py

```diff
@@ -303,7 +304,7 @@ def absolute(self) -> S3Path:
         # We can't compute the absolute path from a relative one
         raise ValueError("Absolute path can't be determined for relative S3Path objects")
 
-    def owner(self) -> str:
+    def owner(self, *, follow_symlinks: bool = True) -> str:
```

Comment:
`follow_symlinks` [was added in Python 3.13](https://docs.python.org/3/whatsnew/3.13.html#pathlib) to `Path.is_file`, `Path.is_dir`, `Path.owner()`, and `Path.group()`. This change makes the signatures compatible with Python 3.13+, but since they're optional keyword-only parameters, they shouldn't disrupt compatibility with older Python versions.

Note, [other places](https://github.com/liormizr/s3path/blob/fdda742f49138ad637be51357d556c2e8e8584b5/s3path/current_version.py#L287-L289) attempt to defensively program around a user who tries to set `follow_symlinks`, but while I didn't remove those checks, I think they're excessive. It's fine to have arguments which are simply ignored.

Reply:
OK
Let's add follow_symlinks as default False - so we won't have behavior change
And add validation because follow_symlinks  is not supported in S3